packages/ember-glimmer/lib/dom.ts

```diff
@@ -1,2 +1,3 @@
+///<reference path="./simple-dom.d.ts" />
```

Comment:
Yup. A tiny hack to make the compiler not complain about the fact that it doesn't know anything about simple-dom.

It was not worth doing anything smarter, because I can see that newer releases of simple-dom already fixed this problem. Landing #15828 will make this go away.